src/components/author/author.scss

```diff
@@ -0,0 +1,13 @@
+@use "../../compiled/tokens/scss/color";
+
+/**
+ * Container
+ */
+
+.c-author {
+  color: color.$base-gray-dark;
+}
+
+.t-dark .c-author {
+  color: color.$base-white;
```

Comment:
Should I add a custom property color for this theme-dependent switch to the themes scss file?

Reply:
What I would do is this:

1. Add a new `--theme-color-text-muted` or `--theme-color-text-subtle` (or whatever) property to `src/base/_themes.scss` for light and dark themes.
1. Update `.c-author` to reference this property.

But I'm open to other ideas.